tokio/CHANGELOG.md

```diff
@@ -3,107 +3,107 @@
 Commit to the API and long-term support.
 
 ### Fixed
-- sync: spurious wakeup in `watch` (#3234).
+- sync: spurious wakeup in `watch` ([#3234]).
```

Comment:
> ([#3234]).

This (and all other links) is displayed like `[#3234]` in markdown. You have to add links to issues/PRs.